nuxt/pages/index.vue

```diff
@@ -251,15 +251,15 @@ en:
   maevsiDescription: maevsi is the manager for events supported by invitees.
   maevsiDescription1: Personal invitations. Proper feedback.
   maevsiDescription2: Don't miss out on events your friends are going to.
-  smartphoneDescription: With meavsi you no longer have to organize your events via chat groups anymore. You communicate all important information clearly, keep control over individual invitations and have your head free for the content of your event – your guests will thank you for it.
-  smartphoneTitle: Do you still organize your event like that?
-  stepCreateDescription: Piece of mind for you and your guests, instead of a bustling group chat
+  smartphoneDescription: With meavsi, you no longer have to organize your events via chat groups anymore. You communicate all important informations clearly, keep control over individual invitations and focus on the content of your event without distractions – your guests will thank you for it.
+  smartphoneTitle: You still organize your event like that?
```

Comment:
Revert

Reply:
Revert what exactly? 255 makes sense. What about 254 and 256?

Reply:
`Do you`